.github/workflows/go.yml

```diff
@@ -46,6 +46,9 @@ jobs:
       - name: Run init
         run: bin/task init
 
+      - name: Regenerate and reformat
+        run: bin/task gen fmt
```

Comment:
To ensure that files are regenerated and formatted. Checked by `git diff --exit-code` below